api/integrations/rudderstack/rudderstack.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import typing
+
+import rudder_analytics
+
+from integrations.common.wrapper import AbstractBaseIdentityIntegrationWrapper
+
+if typing.TYPE_CHECKING:
+    from environments.identities.models import Identity
+    from features.models import FeatureState
+
+logger = logging.getLogger(__name__)
+
+
+class RudderstackWrapper(AbstractBaseIdentityIntegrationWrapper):
+    def __init__(self, base_url: str, api_key: str):
+        rudder_analytics.write_key = api_key
+        rudder_analytics.data_plane_url = base_url
+
+    def _identify_user(self, user_data: dict) -> None:
+        rudder_analytics.identify(**user_data)
+        logger.debug(f"Sent event to Rudderstack.")
+
+    def generate_user_data(
```

Comment:
These should be a test for this method.

Reply:
This is basically just marshalling our model into their required JSON schema. Not sure it's worth testing this or how we would?

Reply:
Just to add, none of the existing analytics integrations have a test for this method

Reply:
Of course it's worth testing and it doesn't matter than none of the others are tested - they should be! 

```python
def test_rudderstack_wrapper_generate_user_data():
    given
    # these should be fixtures but for the purposes of demonstration...
    identity = Identity.objects.create(...)
    feature = Feature.objects.create(...)
    identity_feature_state = FeatureState.objects.create(...)

    wrapper = RudderstackWrapper(...)

    # When
    user_data = wrapper.generate_user_data(identity, feature_states)

    # Then
    assert user_data == {"user_id": identity.identifier, "traits": ...}
```